js/src/Ice/PropertyNames.js

```diff
@@ -46,7 +29,7 @@ PropertyNames.IceProps = [
     new Property("Ice.Admin.Locator.Locator", false, "", false),
     new Property("Ice.Admin.Locator.Router", false, "", false),
     new Property("Ice.Admin.Locator.CollocationOptimized", false, "", false),
-    new Property("^Ice.Admin.Locator.Context..", true, "", false),
+    new Property("^Ice\.Admin\.Locator\.Context\..", true, "", false),
```

Comment:
It's not clear to me why this regeneration added these escapes.

Reply:
The escapes look correct, if I recall correctly Joe committed a fix for the handling of regexp in JavaScript.

Reply:
For what it's worth, the escapes are always added by the script,
but Prettier removed them when JS got reformatted.

Reply:
interesting

Reply:
Actually, if we want to escape the dot, shouldn't it be `\\.`, otherwise `\.` is just.

I guess the best is to create a separate issue, it seems unrelated to the PR.